developers/wcs/tools/collections-tool.mdx

```diff
@@ -1,77 +1,116 @@
 
-To open the query app, connect to your WCD [WCD Dashboard](https://console.weaviate.cloud).
+After you select the collection tool, the `Clusters` panel lists the clusters that are available. Select a cluster from the list to create a new collection or to see the collections that are already in the cluster.
 
-In the left-hand menu, click the Query app icon.
+If you don't see any available clusters, select the `Clusters` option from the left hand menu and follow the steps to [create a new cluster](/developers/wcs/create-instance).
 
-<center><img src={WCDQueryConsoleLocation} width="25%" alt="WCD query app location"/></center>
+<center><img src={WCDEdPick} width="75%" alt="Select cluster"/></center>
 
-## Select a Weaviate instance
+## Create a collection
 
-The query app connects directly to your WCD clusters. The query app can also connect to other Weaviate instances if you have the proper authorization credentials.
+The `Create collection` button opens the collection editor. Click the button to create a new collection.
 
-### Connect to a WCD cluster
+<center><img src={WCDEdColl} width="35%" alt="Create collection"/></center>
 
+### Edit the General section
 
-In this example, the WCD Dashboard is connected to an external instance as well as the two WCD instances.
+Use the `General` section for collection wide settings.
 
-<center><img src={WCDConnectedInstances} width="100%" alt="Connected instances"/></center>
+<center><img src={WCDEdGen} width="75%" alt="General edit section"/></center>
 
-The GraphQL console can query any of the instances.
+#### Name the collection.
 
-To select a cluster, click the `Query` button and chose a cluster from the list that appears.
+The collection name must begin with a capital letter.
 
-### Authentication
+The name can only contain letters, numbers, and the under-score (_) character. Spaces are not allowed.
 
-The query app connects to WCD instances, sandboxes and serverless clusters, without any additional authentication details. The query app can also connect to external Weaviate instances if you have anonymous access enabled or authentication credentials.
+#### Select multi-tenancy
 
-### Pass authentication details
+[Multi-tenant collections](/developers/weaviate/manage-data/multi-tenancy) are useful if you have multiple users or need data isolation. By default, collections are single tenant. Set the `Multi-tenancy` toggle to enable multi-tenancy.
 
-import ExternalAuth from '/_includes/wcs/query-auth-details.mdx';
+#### Set a default vectorizer
 
-<ExternalAuth/>
+Set a [vectorizer](/developers/weaviate/config-refs/schema#vectorizer) for the collection. Weaviate uses the default vectorizer when you import unvectorized data and when Weaviate queries external APIs that require vectorized inputs.
 
-### Pass inference keys
+The `Vectorizer` setting is a drop-down list. After you select a vectorizer, the editor expands to let you configure the model.
```

Comment:
It might be beneficial to point out that the available vectorizer options change based on the chosen vectorizer.